File: 3_recursão/2_saudação.py
def sauda2(nome):
    print('Como vai ' + nome + '?')

def tchau():
    print('Ok, tchau!')

def sauda(nome):
    print('Olá, ' + nome + '!')
    sauda2(nome)
    print('Preparando para dizer tchau...')
    tchau()

print(sauda('Lucas Linyker'))

File: 3_recursão/2_saudação.js
function sauda2(nome) {
    console.log(`Como vai ${nome}?`)
}

function tchau() {
    console.log(`Ok, tchau!`)
}

function sauda(nome) {
    console.log(`Olá, ${nome}!`)
    sauda2(nome)
    console.log(`Preparando para dizer tchau...`)
    tchau(nome)
}

console.log(sauda('Lucas Linyker'))